### ISSUE TITLE
GluegunParameters Import Conflicts with Local Definition

### ISSUE THREAD

**Issue by SgiobairOg** [infinitered/gluegun#664]
## Details
- Version: 4.0.1
- Typescript

## Issue
CLI Builds fail with the following typescript error: 

```bash
node_modules/gluegun/build/types/domain/toolbox.d.ts:5:164 - error TS2440: Import declaration conflicts with local declaration of 'GluegunParameters'.
```

On inspection it looks like there was an intent to move the declaration of 'GluegunParameters' into another file and then import the definitions from 'toolbox' this seems to be the case with other definitions. It looks like that move didn't happen though and the toolbox is importing 'GluegunParameters' from itself?

MIght  be able to fix this and submit a PR but not sure if I'll have a minute or if maybe theres some other issue here. 

Thanks for the great project though. It has been very useful.

**Comment by cmburcus** [comment]
I have the same issue :(

**Comment by mike-morr** [comment]
Anyone find a workaround for this?

**Comment by cmburcus** [comment]
@mike-morr I tried building from the latest version all the way down to version 3.3.3 in typescript with the same error. The only way it works is if you're not using typescript.

I hope this will be fixed soon but it seems odd that it would have gone unnoticed by the devs which makes me thing there's something else going on...

**Comment by mike-morr** [comment]
@jamonholmgren from what I can tell this is fixed in master.  Any chance we can get this released.  It currently is completely broken for TypeScript.  It won't even build the out of the box template.

**Comment by jamonholmgren** [comment]
Hey everyone, sorry, I've been out of town. I'll get a new version deployed very soon. This is a problem which was uncovered by the latest TypeScript and fixed in this commit: bbb5a25bf1939685b9adba21df2c2dd90220eb42

**Comment by SgiobairOg** [comment]
No problem at all. Thanks!

**Comment by jamonholmgren** [comment]
:tada: This issue is fixed in version 4.1.1 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/gluegun)
- [GitHub release](https://github.com/infinitered/gluegun/releases/tag/v4.1.1)

**Comment by cmburcus** [comment]
@jamonholmgren thank you very much!!! :)
